fdbcli/fdbcli.actor.cpp

```diff
@@ -427,6 +429,9 @@ static void printProgramUsage(const char* name) {
 	       "  --log-dir PATH Specifes the output directory for trace files. If\n"
 	       "                 unspecified, defaults to the current directory. Has\n"
 	       "                 no effect unless --log is specified.\n"
+	       "  --log-group LOG_GROUP\n"
```

Comment:
Are you also introducing a change to update these in other binaries?

Reply:
I will create an issue to fix that and hopefully I can follow up on that with a separate PR.